core/controllers/collection_viewer_test.py

```diff
@@ -0,0 +1,191 @@
+# Copyright 2015 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for the page that allows learners to play through a collection."""
+
+__author__ = 'Ben Henning'
+
+from core.controllers import collection_viewer
+from core.domain import collection_domain
+from core.domain import collection_services
+from core.domain import rights_manager
+from core.tests import test_utils
+import feconf
+
+
+class CollectionViewerPermissionsTest(test_utils.GenericTestBase):
+    """Test permissions for learners to view collections."""
+
+    COL_ID = 'cid'
```

Comment:
Actually, scratch this. The more tests I read, the more used I'm getting to COL_ID. I think it's probably fine.